### ISSUE TITLE
Delay rendering directory tree when reloading exported

### ISSUE THREAD

**Issue by pkienzle** [reductus/reductus#100]
Rendering the directory tree requires caching the entire directory on the reductus server. When reloading an exported dataset where the files are not already cached (e.g., for an old dataset or for a test server), there can be a considerable delay before the template is shown as the files are downloaded.

This is not a significant issue since the server will probably have the data already cached for active experiments, so only change it if it is convenient.

**Comment by bmaranville** [comment]
You're right... the delay is annoying.  I'll look into the timing and see how difficult it would be to render the template and data first, and then have the files load in the background.
I think the main reason it waits for the data to reload is so that the state can be properly set for the loader checkboxes after the tree is rendered, and the tree can't render until the metadata for all the files has reloaded, but maybe all it would take is to add another call to the filebrowser renderer after the files are reloaded.
